src/components/POLine/Item/ItemForm.js

```diff
@@ -110,7 +110,9 @@ class ItemForm extends Component {
         change('instanceId', inventoryData.instanceId);
       } else {
         change('instanceId', null);
-        locations.forEach((_, i) => change(`locations[${i}].holdingId`, null));
```

Comment:
`locations?.forEach...` should work here as well.
Thanks @zburke for getting into it.

Reply:
I see that's how it's implemented a few lines later in `setIsPackage` so I will change this line to be consistent with that one. Good thinking; thanks for mentioning it.